code/modules/surgery/experimental_dissection.dm

```diff
@@ -45,31 +45,34 @@
 	user.visible_message("<span class='notice'>[user] starts dissecting [target].</span>", "<span class='notice'>You start dissecting [target].</span>")
 
 /datum/surgery_step/dissection/proc/check_value(mob/living/target, datum/surgery/advanced/experimental_dissection/ED)
-	var/cost = BASE_HUMAN_REWARD
+	var/cost = 0
 	var/multi_surgery_adjust = 0
 
 	//determine bonus applied
+	var/list/mob_mult_list = list(/mob/living/simple_animal/hostile/asteroid/elite = 6,
+									/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient/crystal = 5,
+									/mob/living/simple_animal/hostile/asteroid/basilisk/watcher/forgotten = 3,
+									/mob/living/simple_animal/hostile/asteroid/goliath/beast/ancient = 2
+									)
 	if(isalienqueen(target) || isalienroyal(target))
-		cost = (BASE_HUMAN_REWARD*38)
+		cost = (MAX_DISSECTION_REWARD*38)
 	else if(isalienadult(target))
-		cost = (BASE_HUMAN_REWARD*30)
-	else if(ismonkey(target))
-		cost = (BASE_HUMAN_REWARD*0.5)
+		cost = (MAX_DISSECTION_REWARD*30)
+	else if(isalien(target))
+		cost = (MAX_DISSECTION_REWARD*14)
+	else if(ismegafauna(target))
+		cost = (MAX_DISSECTION_REWARD*30)
 	else if(ishuman(target))
 		var/mob/living/carbon/human/H = target
 		if(H?.dna?.species)
 			if(isabductor(H))
-				cost = (BASE_HUMAN_REWARD*24)
+				cost = (MAX_DISSECTION_REWARD*24)
 			else if(iszombie(H) || isshadow(H) || isandroid(H))
-				cost = (BASE_HUMAN_REWARD*20)
-			else if(isjellyperson(H) || ispodperson(H) || isalien(H))
-				cost = (BASE_HUMAN_REWARD*14)
-			else if(isskeleton(H))
-				cost = (BASE_HUMAN_REWARD * 0.5)
-	else
-		cost = (BASE_HUMAN_REWARD * 0.5)
-
-
+				cost = (MAX_DISSECTION_REWARD*20)
+	else for(var/type in mob_mult_list) // THIS. ELSE IF(). ENDS. HERE.
```

Comment:
I think you can do this without iterating:
`if(target in mob_multi_list)
     cost = [...]`

Reply:
you wouldn't get subtypes, I'm pretty sure, but if that's not an issue it's possible yeh

Reply:
Directly comparing types won't get subtypes, though the only thing that's using that right now is elites.